src/site/xdoc/checks/misc/translation.xml.template

```diff
@@ -114,17 +126,29 @@ messages.translations: Properties file 'messages_fr.translations' is missing.
         </p>
         <macro name="example">
           <param name="path"
-                 value="resources/com/puppycrawl/tools/checkstyle/checks/translation/Example2.txt"/>
+                 value="resources/com/puppycrawl/tools/checkstyle/checks/translation/Example2.java"/>
           <param name="type" value="config"/>
         </macro>
         <p id="Example2-code">
-          Example:
+          Example2:
```

Comment:
please apply print of folder content to all Examples.
Print of files in folder can be placed to Input file and loaded to template by "raw" type macros

see example at https://github.com/checkstyle/checkstyle/commit/d2da459f3e1de3464160e308b82abaf7b58c8e0c#diff-6cc6ac93b182101bb7cbcffff9f97f80a68b7a8df13483dac2ec65db7cc3b3fbR9

in this case Input file has config and visual representation of validation, it is easier to maintain.